.github/workflows/manual-build-artifacts.yml

```
@@ -0,0 +1,236 @@
+name: Manual Firmware Build (Artifacts Only)
+
+on:
+  workflow_dispatch:
```

actionlint (1.7.9)

maximum number of inputs for "workflow_dispatch" event is 10 but 14 inputs are provided. see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs

(events)